python_scripts/replace_template.py

```diff
@@ -59,8 +72,23 @@ def main():
             pywikibot.output('Page %s not found' % page.title(as_link=True))
             continue
 
-        new_text = original_text.replace(
-            '{{' + old_template, '{{' + new_template)
+        if parameter:
+            # Only replace templates with a parameter if given.
+            parts = original_text.split('{{' + old_template)
+            new_text = parts[0]
+            for part in parts[1:]:
+                if re.search(
```

Comment:
You'll want to ensure there is nothing before or after the parameter parts. as it is `parameter = (a, 1)` will give false positives for both of the below:
`{{old|a=11|}`
`{{old|a=2|aaa=1}}`